Library/Formula/gexiv2.rb

```diff
@@ -14,11 +14,14 @@ class Gexiv2 < Formula
   depends_on "pkg-config" => :build
   depends_on "glib"
   depends_on "exiv2"
+  depends_on "gobject-introspection"
```

Comment:
Can you make this a build dependency?

(Also, nit: can you sort the :build dependencies first?)